back/src/test/kotlin/com/sos/smartopenspace/controllers/TalkControllerTest.kt

```diff
@@ -64,6 +64,29 @@ class TalkControllerTest {
         ).andExpect(MockMvcResultMatchers.status().isBadRequest)
     }
 
+    @Test
+    fun `Asking for an specific talk returns an ok status`() {
+        val organizer = anySavedUser()
+        val talk = anySavedTalk()
+        userRepository.save(anyUser(talk))
+        val room = anySavedRoom()
+        openSpaceRepository.save(anyOpenSpaceWith(talk, organizer, room))
+
+        mockMvc.perform(
+            MockMvcRequestBuilders.get("/talk/${talk.id}")
+        ).andExpect(MockMvcResultMatchers.status().isOk)
+            .andExpect(MockMvcResultMatchers.jsonPath("$.id").value(talk.id))
+            .andExpect(MockMvcResultMatchers.jsonPath("$.name").value(talk.name))
+    }
+
+    @Test
+    fun `Asking for a talk that not exist returns a bad request`() {
+        mockMvc.perform(
+            MockMvcRequestBuilders.get("/talk/-1")
```

Comment:
esto por ahi convenga testearlo con un numero positivo, porque me deja la duda de si el error es porque es negativo o porque no existe en bd.

Reply:
Por qué piensan que es un bad request? Yo creo que debería retornar un 404, not found (recién miré [este post](https://stackoverflow.com/questions/11746894/what-is-the-proper-rest-response-code-for-a-valid-request-but-an-empty-data), parece q hay bastantes posturas...)